### PR TITLE
added level #2  HTML

### LINKED ISSUE #23274: Stacking context example 3, stacking context hierarchy misinformation
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_context_example_3

The paragraph listing the stacking context hierarchy shows three or more (three dots) LEVEL#2 items, while in the example there are only two. This makes the example hard to understand.

```
Root stacking context
  LEVEL #1
    LEVEL #2 (z-index: 1)
      LEVEL #3
      …
      LEVEL #3
    LEVEL #2 (z-index: 1)
    …
    LEVEL #2 (z-index: 1)
  LEVEL #1
  …
  LEVEL #1
```

image

### PR DESCRIPTION
**Pull request by TjasaZil**
### Description

I added a few more level 2 divs, in order to better illustrate the example.

### Motivation

The example is more understandable, based on the description above the code.

### Related issues and pull requests

Fix #23274

### CODE REVIEW

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hello @zuzexx, the author in the issue is referring to the list in the [Description](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_context_example_3#description) section, under the sentence "To better understand the situation, here is the stacking context hierarchy:". This list contains the three dots for illustration purposes.

Even if you try to improve the code snippets, I believe the list with three dots, which is causing the confusion, still exists.

Could you make corresponding updates to the list to remove the three dots and also to match your new example?

**Comment by zuzexx** [comment]
I can do the changes. Just that I understand correctly, you want me to change the three dots between level 2 divs to the appropriate amount of divs so that it reflects the new example?

**Comment by dipikabh** [comment]
> I can do the changes. Just that I understand correctly, you want me to change the three dots between level 2 divs to the appropriate amount of divs so that it reflects the new example?

Yes, that's correct!

File: files/en-us/web/css/css_positioning/understanding_z_index/stacking_context_example_3/index.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ To better understand the situation, here is the stacking context hierarchy:
       - LEVEL #3
 
     - LEVEL #2 (`z-index`: 1)
-    - …
+    - LEVEL #2 (`z-index`: 1)
```
Comment (dipikabh) [inline files/en-us/web/css/css_positioning/understanding_z_index/stacking_context_example_3/index.md:41]:
```suggestion

```
let's keep the ellipsis

File: files/en-us/web/css/css_positioning/understanding_z_index/stacking_context_example_3/index.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ To better understand the situation, here is the stacking context hierarchy:
...
     - LEVEL #2 (`z-index`: 1)
 
   - LEVEL #1
-  - …
+  - LEVEL #1
```
Comment (dipikabh) [inline files/en-us/web/css/css_positioning/understanding_z_index/stacking_context_example_3/index.md:45]:
```suggestion

```
let's keep the ellipsis

**Review by dipikabh — CHANGES_REQUESTED** [review]
Perhaps, we don't need to remove the ellipsis. We can clarify the use of ellipsis in the description.

How about updating the following line as:

line 28: "To better understand the situation, the following example illustrates the stacking context hierarchy (the three dots "..." represent multiple repetition of the previous line):

File: files/en-us/web/css/css_positioning/understanding_z_index/stacking_context_example_3/index.md (hunk 2 above)
Comment (zuzexx) [reply files/en-us/web/css/css_positioning/understanding_z_index/stacking_context_example_3/index.md:45]: Ok, thank you for clarification. I will get to it once I come home.

**Review by dipikabh — APPROVED** [review]
Thank you, @zuzexx! 👍

**Pull request merged**